### PR TITLE
Fixed Issue #100 For Hacktoberfest

### LINKED ISSUE #100: 📝. Steps of an Election
This is currently highlighted at the lower level in `electionguard-python` docs but should get migrated to this repository. 

See: https://electionguard-python.readthedocs.io/en/latest/

Pull from the step by step process in the python repository and create a non code version of these steps. This should highlight the basics of setting up and running an election. 

There are some temporary pages to help determine where this should be added
All the pages needed to be added to `docs/basics`

### PR DESCRIPTION
**Pull request by KanhaKorgaonkar**
[//]: # (🚨 Please review the CONTRIBUTING.md in this repository. 💔Thank you!)

### Issue
Fixes #100 

### Description
I completed the tasks in #100, and updated the following files.
- 4_Decryption.md : Added information related to the decrypt step, and an image from the Undraw folder
- 5_Publish_Record.md : Added information related to the publish step, and an image.
- 6_Verification.md : Added information related to the verify step, and an image.
- 3_Cast_and_Challenge.md : Added an image.

### Testing
All documentation based changes.

### CODE REVIEW

**Comment by KanhaKorgaonkar** [comment]
Hi Maintainers, 
I am a first time contributor, so please pardon any mistakes I made with the PR message above. Would also welcome advice regarding how to be a better open source contributor. Thanks! Have a nice day. 👍

**Comment by keithrfung** [comment]
Will review in the morning but welcome @KanhaKorgaonkar

File: docs/basics/steps/3_Cast_and_Challenge.md [hunk 1]
```diff
@@ -2,4 +2,9 @@
 
 Each ballot that is completed by a voter must be either cast or challenged. A cast ballot is a ballot that the voter accepts as valid and wishes to include in the official election tally. A challenge ballot, also referred to as a spoiled ballot, is a ballot that the voter does not accept as valid and wishes to exclude from the official election tally.
 
-ElectionGuard includes a mechanism to mark a specific ballot as either cast or spoiled. Cast ballots are included in the tally record, while spoiled ballots are not. Spoiled ballots are decrypted into plaintext and published along with the rest of the election record.
\ No newline at end of file
+![Encrypt][encrypt-image]
+
+<!-- Links -->
```
Comment (keithrfung) [inline docs/basics/steps/3_Cast_and_Challenge.md:7]: Please put the links at the bottom of the page. I'm gonna have @rc_ms review this for content but this looks like a good start.

Comment (KanhaKorgaonkar) [reply docs/basics/steps/3_Cast_and_Challenge.md:7]: Okay, I will do that as soon as possible! Thanks for your feedback.

Comment (KanhaKorgaonkar) [reply docs/basics/steps/3_Cast_and_Challenge.md:9]: Well, @keithrfung I put the links at the bottom of the page. Let me know if this is fine.

Comment (keithrfung) [reply docs/basics/steps/3_Cast_and_Challenge.md:9]: Awesome this looks good. @rc-ms will review the content. Probably on Monday but thanks for your contribution.

**Review by rc-ms — APPROVED** [review]
Thanks for these submissions!  Appreciate your contribution

**Pull request merged**